trace/doc.go

```diff
@@ -0,0 +1,164 @@
+// Package trace provies an API for initiating and reporting SSF
+// traces and attaching spans to them. Veneur's tracing API also
+// provides an experimental opentracing compatibility layer.
+//
+// The Veneur tracing API is completely independent of the opentracing
+// compatibility layer, with the exception of the function
+// StartSpanFromContext.
+//
+// Setup and Structure of the API
+//
+// To use the API, user code must set its service name via the global
+// variable Service, which is used by trace visualization tools to
+// tell apart the spans reported by different services. Code using the
+// trace API must set its Service, and set it only once, in main():
+//
+//    trace.Service = "my-awesome-service"
+//
+// The trace API is built on three abstractions: the
+// Client, the ClientBackend and the Span.
+//
+// Context
+//
+// Since it is very common for networked code to use context.Context,
```

Comment:
I know this is implied, but I'd add "from the Go standard library" or something like that, because the word `context` is already overloaded in the Go project, even before we introduce the opentracing concept. Just including it the first time we introduce the term (here) should be sufficient to resolve any ambiguity about what we mean going forward.